UKHomeOffice/ide-controller
# COP 1976: Split table component

## Description
This branch splits the information table found to the left of the UI and updates formatting to align with the latest prototype.

## To test
- Install app and run dev (this will open an electron app window)
- If this page is blank, refresh the page with cmd + r (mac) or ctl + r (windows)
- The table to the left of the UI should appear as two separate tables. 

## Developer Checklist

\* Required
